src/index.html

```diff
@@ -11,8 +11,46 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), stylesheets with .scss extension should not be linked directly in HTML. You should compile them to CSS and then link the CSS file.